src/main/pages/setup-openshift/openshift-config.md

```diff
@@ -34,15 +34,52 @@ You can manually convert properties into envs, just make sure to follow [instruc
 
 To enable https for server and workspace routes, follow instructions in [setup docs single user][openshift-single-user] and [multi-user][openshift-multi-user]. To migrate an existing Che deployment to https, do the following:
 
-<span style="color:red;">IMPORTANT!</span> Self-signed certificates aren't acceptable.
-
 1. Update Che deployment with `PROTOCOL=https, WS_PROTOCOL=wss, TLS=true`
 2. Manually edit or recreate routes for Che and Keycloak `oc apply -f https`
 3. Once done, go to `https://keycloak-${NAMESPACE}.${ROUTING_SUFFIX}`, log in to admin console.
 Default credentials are `admin:admin`.
 Go to Clients, `che-public` client and edit **Valid Redirect URIs** and **Web Origins** URLs so that they use **https** protocol.
 You do not need to do that if you initially deploy Che with https support.
 
+## HTTPS Mode - Self-Signed Certs
+
+If you enable HTTPS mode for Che on an OpenShift installation that does not have certificates signed by a public authority, it won't be possible to start workspaces or even login, if it's a multi user Che deployment. There is a lot of communication between Che server and workspace agents, Che server and Keycloak. Therefore, self signed certs should be added to Java trust store of Che server and Keycloak (only for a multi user Che deployment) pods, as well as workspace images. While there is automation for Che server and Keycloak, certs should be manually added to workspace images, since adding a root certificate requires sudo privileges which an arbitrary OpenShift user may not have.
```

Comment:
You can change first sentence to: `If you enable HTTPS mode for multi-user Che on an OpenShift installation that does not have certificates signed by a public authority, it won't be possible to start workspaces or even login.`

Also is this true of k8s as well?

Reply:
Changed.

And these instructions work for OpenShift only, as OpenShift cert is used (the one that gets generated during installation)